crowbar_framework/app/models/batch/export.rb

```diff
@@ -46,7 +46,7 @@ def filename
 
     def persist!
       path.open("w") do |file|
-        file.write content.stringify_keys(true).to_yaml
+        file.write content.stringify_keys.to_yaml
```

Comment:
See http://apidock.com/rails/v4.2.1/Hash/stringify_keys for this but in short the true was just wrong due to stringify_keys doesn't take arguments.